node/node_test.go

```diff
@@ -375,6 +378,120 @@ func TestMaxProposalBlockSize(t *testing.T) {
 	assert.EqualValues(t, partSet.ByteSize(), int64(pb.Size()))
 }
 
+func TestMaxProposalBlockSize(t *testing.T) {
+	config := cfg.ResetTestRoot("node_create_proposal")
+	defer os.RemoveAll(config.RootDir)
+	cc := proxy.NewLocalClientCreator(kvstore.NewApplication())
+	proxyApp := proxy.NewAppConns(cc)
+	err := proxyApp.Start()
+	require.Nil(t, err)
+	defer proxyApp.Stop() //nolint:errcheck // ignore for tests
+
+	logger := log.TestingLogger()
+
+	state, stateDB, _ := state(types.MaxVotesCount, int64(1))
+	stateStore := sm.NewStore(stateDB)
+	var maxBytes int64 = 1024 * 1024 * 2
+	state.ConsensusParams.Block.MaxBytes = maxBytes
+	proposerAddr, _ := state.Validators.GetByIndex(0)
+
+	// Make Mempool
+	memplMetrics := mempl.PrometheusMetrics("node_test_2")
+	mempool := mempl.NewCListMempool(
+		config.Mempool,
+		proxyApp.Mempool(),
+		state.LastBlockHeight,
+		mempl.WithMetrics(memplMetrics),
```

Comment:
```suggestion
		mempl.WithMetrics(mempl.NopMetrics()),
```
to avoid the panic in ci

Reply:
I've made the change for the other tests as well